Description: Fix spelling errors.
 allow to <verb> -> allows <verb>ing
 calulated       -> calculated
 correspoding    -> corresponding
 specifed        -> specified
Author: Bas Couwenberg <sebastic@debian.org>

--- a/gui/wxpython/gmodeler/g.gui.gmodeler.html
+++ b/gui/wxpython/gmodeler/g.gui.gmodeler.html
@@ -81,7 +81,7 @@ color="red">16</font>) Show manual and l
 There is also lower menu bar in Graphical modeler dialog where one can
 manage model items, see commands, add or manage model variables,
 define default values and descriptions. Python editor dialog window
-allows to see performation written in Python code. The rightmost tab of
+allows seeing performation written in Python code. The rightmost tab of
 the bottom menu is automatically triggered when the model is activated and
 shows all the steps of running GRASS modeler modules. In case of errors
 in the calculation process, it is written at that place.
--- a/lib/gis/lz4.h
+++ b/lib/gis/lz4.h
@@ -119,7 +119,7 @@ int LZ4_compressBound(int inputSize);
 
 /*
 LZ4_compress_fast() :
-    Same as LZ4_compress_default(), but allows to select an "acceleration" factor.
+    Same as LZ4_compress_default(), but allows selecting an "acceleration" factor.
     The larger the acceleration value, the faster the algorithm, but also the lesser the compression.
     It's a trade-off. It can be fine tuned, with each successive value providing roughly +~3% to speed.
     An acceleration value of "1" is the same as regular LZ4_compress_default()
--- a/raster/r.neighbors/r.neighbors.html
+++ b/raster/r.neighbors/r.neighbors.html
@@ -389,7 +389,7 @@ g.region rows=10 cols=10
 </pre></div>
 
 Fill 50% of computational region with randomly located cells.
-"distance=0" will allow to fill adjacent cells.
+"distance=0" will allow filling adjacent cells.
 <div class="code"><pre>
 r.random.cells output=random_cells distance=0 ncells=50
 </pre></div>
--- a/raster/r.solute.transport/r.solute.transport.html
+++ b/raster/r.solute.transport/r.solute.transport.html
@@ -60,7 +60,7 @@ The calculation and boundary status of s
 The following states are supportet:
 
 <ul>
-<li>0 == inactive - the cell with status 0 will not be calulated, active cells will have a no flow boundary to an inactive cell</li>
+<li>0 == inactive - the cell with status 0 will not be calculated, active cells will have a no flow boundary to an inactive cell</li>
 <li>1 == active - this cell is used for sloute transport calculation, inner sources can be defined for those cells</li>
 <li>2 == Dirichlet - cells of this type will have a fixed concentration value which do not change over time </li>
 <li>3 == Transmission - cells of this type should be placed on out-flow boundaries to assure the flow of the solute stream out </li>
--- a/raster3d/r3.gwflow/r3.gwflow.html
+++ b/raster3d/r3.gwflow/r3.gwflow.html
@@ -47,7 +47,7 @@ status of single cells can be set with t
 cell states are supported:
 
 <ul>
-<li>0 == inactive - the cell with status 0 will not be calulated,
+<li>0 == inactive - the cell with status 0 will not be calculated,
  active cells will have a no flow boundary to an inactive cell</li>
 <li>1 == active - this cell is used for groundwater calculation,
  inner sources can be defined for those cells</li>
--- a/temporal/t.rast.algebra/t.rast.algebra.html
+++ b/temporal/t.rast.algebra/t.rast.algebra.html
@@ -246,7 +246,7 @@ compared by logical operators and evalua
 the related maps.
 For complex relations the comparison operator can be used to combine conditions: 
 <br>
-The structure is similar to the select operator with the extention of an aggregation operator: 
+The structure is similar to the select operator with the extension of an aggregation operator: 
 {"comparison operator", "topological relations", aggregation operator, "temporal operator"}
 <br>
 This aggregation operator (| or &) define the behaviour if a map is related the more 
--- a/vector/v.out.ogr/export_areas.c
+++ b/vector/v.out.ogr/export_areas.c
@@ -77,7 +77,7 @@ int export_areas_single(struct Map_info
                        * not labeled */
         }
         
-        /* find correspoding area */
+        /* find corresponding area */
         area = Vect_get_centroid_area(In, centroid);
         if (area == 0)
             continue;
@@ -184,7 +184,7 @@ int export_areas_multi(struct Map_info *
                                 "Category %d not found in line %d, field %d"),
                               cat, line, field);
             
-            /* find correspoding area */
+            /* find corresponding area */
             area = Vect_get_centroid_area(In, line);
             if (area == 0)
                 continue;
@@ -241,7 +241,7 @@ int export_areas_multi(struct Map_info *
                        * not labeled */
         }
 
-        /* find correspoding area */
+        /* find corresponding area */
 	line = Vect_get_next_line_id(In);
         area = Vect_get_centroid_area(In, line);
         if (area == 0)
--- a/vector/v.what.rast/v.what.rast.html
+++ b/vector/v.what.rast/v.what.rast.html
@@ -6,7 +6,7 @@ vector attribute table with the retrieve
 
 <p>The column type needs to be numeric (integer, float, double,
 ...). If the column doesn't exist in the vector attribute table than
-the module will create the new column of type correspoding with the
+the module will create the new column of type corresponding with the
 input raster map.
 
 <p>
--- a/general/g.proj/datumtrans.c
+++ b/general/g.proj/datumtrans.c
@@ -91,7 +91,7 @@ int set_datum(char *datum)
  *        co-ordinate system definition
  * 
  * \param datumtrans  Index number of parameter set to use, 0 to leave
- *                    unspecifed (or remove specific parameters, leaving just
+ *                    unspecified (or remove specific parameters, leaving just
  *                    the datum name), -1 to list the available parameter
  *                    sets for this datum and exit.
  *
--- a/gui/wxpython/docs/wxGUI.modules.html
+++ b/gui/wxpython/docs/wxGUI.modules.html
@@ -93,7 +93,7 @@ For raster, vector or 3D raster input, t
 widget which after clicking on the arrow to the right pops up
 a list of existing maps from different mapsets.
 Selecting a map from the popup list will add it to the entry field.
-In case multiple maps can be specifed (denoted by <i>[multiple]</i> label),
+In case multiple maps can be specified (denoted by <i>[multiple]</i> label),
 selecting a map from the popup list will append the map names with
 comma in between.
 
--- a/lib/proj/convert.c
+++ b/lib/proj/convert.c
@@ -267,7 +267,7 @@ OGRSpatialReferenceH GPJ_grass_to_osr(co
  * \param hSRS        OGRSpatialReferenceH object containing the co-ordinate 
  *                    system to be converted
  * \param datumtrans  Index number of datum parameter set to use, 0 to leave
- *                    unspecifed
+ *                    unspecified
  * 
  * \return            2 if a projected or lat/long co-ordinate system has been
  *                    defined
@@ -685,7 +685,7 @@ int GPJ_osr_to_grass(struct Cell_head *c
  * \param wkt         Well-known Text (WKT) description of the co-ordinate 
  *                    system to be converted
  * \param datumtrans  Index number of datum parameter set to use, 0 to leave
- *                    unspecifed
+ *                    unspecified
  * 
  * \return            2 if a projected or lat/long co-ordinate system has been
  *                    defined
